forge-game/src/main/java/forge/game/spellability/SpellAbility.java

```diff
@@ -1166,7 +1169,7 @@ public SpellAbility copy(Card host, Player activ, final boolean lki) {
 
             // don't use setHostCard to not trigger the not copied parts yet
 
-            copyHelper(clone, host);
+            copyHelper(clone, host, lki || keepTextChanges);
```

Comment:
If it's lki we should always keep textChanges imo -
fixes this scenario:
1. alter Human type of _Jirina, Dauntless General_
2. use _Quicksilver Elemental_ on her
3. activate new ability -> it should affect the new type of 1.